lib/logster/logger.rb

```diff
@@ -57,7 +57,7 @@ def level
       (ol && ol[Thread.current.object_id]) || @level
     end
 
-    def add_with_opts(severity, message, progname, opts=nil, &block)
+    def add_with_opts(severity, message, progname=progname(), opts=nil, &block)
```

Comment:
per https://ruby-doc.org/stdlib-2.4.0/libdoc/logger/rdoc/Logger.html#method-i-add

the interface could be `Logger#add(severity, message = nil, progname = nil) { ... }`


```ruby
               # File logger.rb, line 454
def add(severity, message = nil, progname = nil)
```